src/DynamoApplications/PathResolvers.cs

```diff
@@ -89,18 +89,21 @@ public CLIPathResolver(string preloaderLocation)
 
             additionalNodeDirectories = new List<string>();
 
-            //this is a subset of libraries loaded for sandbox
             preloadedLibraryPaths = new List<string>
             {
                 "VMDataBridge.dll",
                 "ProtoGeometry.dll",
                 "DesignScriptBuiltin.dll",
                 "DSCoreNodes.dll",
+                "DSOffice.dll",
```

Comment:
Practically speaking, I see no difference in execution time when running a small graph, it is ~5s. I do see a very small difference when using the profiler. Start to finish using the same graph was 15.4s on average before and 15.9s now. So about 0.5s slower when running in the profiler. So we are talking about a few tenths of a second per run in a normal environment.

Reply:
what might be an average number of serial runs required for an optimization?

Reply:
Hard to tell. It could be hundreds. But the graphs are usually much larger than this so the time saved compared to the overall time will be even less significant.